plugins/EpadDB.js

```diff
@@ -9575,6 +9859,12 @@ async function epaddb(fastify, options, done) {
                 ADD COLUMN IF NOT EXISTS maxruntime int(10) AFTER memory;`,
               { transaction: t }
             );
+
+            await fastify.orm.query(
+              `ALTER TABLE plugin_queue
+              MODIFY COLUMN status varchar(10) ;`,
```

Comment:
model says 8

Reply:
status info text has longer char set now so instead of editing model data size we alter the model but the wrong part is that the model needs to be 7 from the previous branch. varchar(10) is correct